src/components/Grid/Grid.css

```diff
@@ -1,3 +1,9 @@
 .pie { background-color: red; }
-.graph { background-color: blue; }
+.graph { background-color: white; }
 .stats { background-color: green; }
+.box
```

Comment:
I would change `.box` to `grid > div`, this way you dont need to pass a box class to each div in the grid.